src/e2e-app/example-viewer/example-viewer.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+import {Component, ElementRef, Injector, Input, OnInit} from '@angular/core';
+import {createCustomElement} from '@angular/elements';
+import {EXAMPLE_COMPONENTS} from '@angular/material-examples';
+
+@Component({
+  selector: 'example-viewer',
+  template: `
+    <div class="label" *ngIf="showLabel">
+      <span class="title"> {{title}} </span>
+      <span class="id"> <{{id}}> </span>
+    </div>
+
+    <div *ngIf="!id">
+      Could not find example {{id}}
+    </div>
+  `,
+  styles: [`
+    .label {
```

Comment:
Could we prefix these classes with `e2e-`?

Reply:
I copied this from Andrew's example viewer in the dev-app, it looks like we don't need any of the styles for the e2e app so I removed them